backend/schemas/user.py

```
@@ -1,8 +1,62 @@
+    profile_picture: Optional[str]  # File path stored in DB
+
+# ✅ User Response Schema (For Auth & Profile APIs)
+class UserResponse(BaseModel):
+    id: int
+    username: str
+    email: str
+    is_active: bool
+    profile_picture: Optional[str] = None
+    university_name: Optional[str] = None
+    department: Optional[str] = None
+    fields_of_interest: List[str] = []  # Default empty list
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)